src/HTML5/Parser/Tokenizer.php

```diff
@@ -1188,4 +1188,18 @@ protected function decodeCharacterReference($inAttribute = false)
 
         return '&';
     }
+
+    /**
+     * Checks whether a (single-byte) character is an ASCII letter or not.
+     *
+     * @param string $input A single-byte string
+     *
+     * @return bool True if it is a letter, False otherwise
+     */
+    protected function is_alpha($input)
+    {
+        $code = ord($input);
+
+        return ($code >= 97 && $code <= 122) || ($code >= 65 && $code <= 90);
```

Comment:
to import readability of the code, you might replace the numbers with `\ord('A')` and so on. PHP optimizes such calls (qualified calls to `ord` with a static string as argument) at compile time, so the runtime is the same than putting the number directly. But it makes it easier to understand what the code is about (as it avoids magic numbers)

Reply:
:+1: